openfl/interface/model.py

```diff
@@ -93,17 +97,7 @@ def save_(
     model_protobuf_path,
     output_filepath,
 ):
-    """Save the model in native format (PyTorch / Keras).
-
-    Args:
-        context (click.core.Context): Click context.
-        plan_config (str): Federated learning plan.
-        cols_config (str): Authorized collaborator list.
-        data_config (str): The data set/shard configuration file.
-        model_protobuf_path (str): The model protobuf to convert.
-        output_filepath (str): Filename the model will be saved to in native
-            format.
-    """
+    """Saves a model proto in framework native format (PyTorch/Keras)."""
```

Comment:
Do you think we should omit naming any specific framework? 

```suggestion
    """Saves a model proto in framework native format."""
```

Reply:
Accepted.